### DISCUSSION TITLE
EE6 - RTL Support?

### DISCUSSION THREAD

**Discussion by mustafa-online** [ExpressionEngine/ExpressionEngine#2925] [Feature Requests]
I wanna know how to add **RTL** support for ee6 admin dashboard? may be an option based on the language? I think it's hardcoded as an LTR.

**Comment by intoeetive** [comment]
Unfortunately currently RTL language direction is not really supported in EE control panel.
We will need to work on it as part of overall internationalization improvement, but we don't have any timeline on that yet
